src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,110 @@
+"""Create a robot template for purpose of syncing subclassof relations.
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/92
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/363
+"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Any, Dict, List, Union
+
+# todo: clear unused imports
+import curies
+import pandas as pd
+from oaklib import get_adapter
+from oaklib.interfaces.basic_ontology_interface import RELATIONSHIP
+from oaklib.types import CURIE, URI
+
+from src.scripts.utils import remove_angle_brackets
+
+# from sssom.util import is_curie
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+ONTOLOGY_DIR = SRC_DIR / 'ontology'
+DEFAULT_OUTPATH = ONTOLOGY_DIR / 'reports' / 'sync-subclassof.robot.template.tsv'
+DEFAULT_ONTOLOGY_DB_PATH = ONTOLOGY_DIR / 'tmp' / 'merged.db'
+ROBOT_ROW = {
+    'term_id': 'ID',
+    'parent_id': 'SC % SPLIT=|',
+    # todo: PR comment to remind about better property for:
+    'xref_ids': 'A oboInOwl:hasDbXref SPLIT=|',  # todo: Nico may provide different value
+}
+# https://curies.readthedocs.io/en/latest/tutorial.html#loading-a-pre-defined-context
+CONV = curies.get_monarch_converter()
```

Comment:
## Unrecognized URI stems & solutions
I decided to try `curies.get_monarch_converter()` first rather than reach for `config/prefixes.csv` and convert it to `yaml` to pass to `curies.Converter.from_prefix_map()`. However I found this set of unrecognized URI stems:

<details><summary>8 unrecognized URI stems</summary>
<p>
```
http://apps.who.int/classifications/icd10/browse/2010/en#/
http://identifiers.org/meddra/
http://identifiers.org/medgen/
http://identifiers.org/mesh/
http://identifiers.org/snomedct/
https://icd.codes/icd10cm/
https://omim.org/entry/
https://omim.org/phenotypicSeries/
```
</p>
</details> 

Of these 8, only 1 of them is in `config/prefixes.csv`: `https://omim.org/entry/`

Proposed solutions:
1. Open an issue in `curies` to try to get these updated.
2. For now, use update `config/prefixes.csv` and use that.

@matentzn I'm going to do (2) and hold off on your word for (1). What do you think?
Also, is it required that my robot template only have CURIEs? Or is it OK if there are also URIs in there?

Reply:
1. `curies` is just the package, if anything, you would have to update bioregistry (which you could, perhaps, at least for some of them).
2. You can use the the "chaining" functionality in curies and combine the obo converter with a custom one (see curies docs)

> Also, is it required that my robot template only have CURIEs? Or is it OK if there are also URIs in there?

It is OK to have URIs

Reply:
Actually, implementation has changed since I wrote this comment, and I'm not using the `curies` package. I think I mainly can't use that package because I need to not be using a general `prefix_map`, but what I mainly need is the list of prefixes for each given source. Right now the prefix maps need to be updated in 2 places (both for this PR's purposes and in general):
i. `prefixes.csv`: For our SemSQL related stuff. However I did open that issue to create this file from a `Converter` object ([Converter -> prefixes.csv exporter #8](https://github.com/monarch-initiative/mondolib/issues/8)).
ii. `metadata/SOURCE.yml` files

If you can think of a way in which `curies` and `Converter` have some role to play in this work (other than [Converter -> prefixes.csv exporter #8](https://github.com/monarch-initiative/mondolib/issues/8)), let me know. If so, then as you suggest above I would probably both use (2) chaining, and (1) open a BioRegistry issue.

Reply:
Please make an issue on sorting out prefix handling. Everything should be using the curies package, and there should be exactly one source of truth for prefixes in mondo-ingest (right now I can count 4).

Reply:
Created new issue:
- #146